webapp/advantage/views.py

```diff
@@ -471,6 +472,22 @@ def cancel_advantage_subscriptions(**kwargs):
     return flask.jsonify(purchase), 200
 
 
+@advantage_decorator(permission="user", response="json")
+@use_kwargs(post_offer_schema, location="json")
+def post_offer(**kwargs):
+    account_id = kwargs.get("account_id")
+    offer_id = kwargs.get("offer_id")
+    marketplace = kwargs.get("marketplace", "canonical-ua")
+
+    g.api.purchase_from_marketplace(
```

Comment:
missing a `return` statement?

Reply:
I swear I added it when I QAed my endpoint. But then I did not commit it!!!